apps/snitch_core/priv/repo/seed/seeds.exs

```diff
@@ -12,8 +10,16 @@ alias Snitch.Seed.{CountryState, PaymentMethods, Orders, Users}
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
 
+alias Snitch.Repo
+alias Snitch.Seed.{CountryState, PaymentMethods, Orders, Users, Stocks}
+
+variant_count = 9
+
```

Comment:
why `9`? what is so cool about it? why do you hate `11` so much in the name of using a `variable`?